gnpy/core/equipment.py

```diff
@@ -134,7 +138,8 @@ def from_json(cls, filename, **kwargs):
         config = Path(filename).parent / 'default_edfa_config.json'
 
         type_variety = kwargs['type_variety']
-        type_def = kwargs.get('type_def', 'variable_gain') #default compatibility with older json eqpt files
+        type_def = kwargs.get('type_def', 'variable_gain')
+        #default compatibility with older json eqpt files
```

Comment:
I think that it's more intuitive if these comments are listed at the same line, or *before* the line they refer to. I've cleaned this.